crates/sequencer_proof_client/src/main.rs

```diff
@@ -52,6 +52,9 @@ impl Cli {
 enum Commands {
     /// Picks the next FRI proof job from the sequencer; sequencer marks job as picked (and will not give it to other clients, until the job expires)
     PickFri {
+        /// VK hash for the proof to be picked
+        #[arg(short, long, value_name = "VK_HASH")]
```

Comment:
This is slightly unergonomic. Some alternatives here:
1. don't pass it all and infer it from SupportedProtocolVersions; you can request all versions until one matches (or exhausted them all)
2. modify the server to support fetching without Protocol Version; against it as it would either add complexity to server API or add space for potential bugs

1 is a neat compromise, but outside the scope of this PR.

Applies across the board.